### PR TITLE
st1 [2169][ADD] payment_acquirer_partner_link, website_sale_partner_acquirer

### PR DESCRIPTION
**Pull request by yostashiro**
[2169](https://www.quartile.co/web?debug=1#id=2169&action=771&model=project.task&view_type=form&menu_id=505)

This PR should be followed by another one that adjusts the invoice print template.

### CODE REVIEW

**Review by kakurai8 — APPROVED** [review]
LGTM

File: payment_acquirer_partner_link/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Quartile Limited
```
Comment (tack-yabeQuartile) [inline payment_acquirer_partner_link/models/account_invoice.py]:
```suggestion
# Copyright 2021-2022 Quartile Limited
```

File: payment_acquirer_partner_link/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Quartile Limited
```
Comment (tack-yabeQuartile) [inline payment_acquirer_partner_link/__manifest__.py]:
```suggestion
# Copyright 2021-2022 Quartile Limited
```

File: payment_acquirer_partner_link/models/payment_acquirer.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Quartile Limited
```
Comment (tack-yabeQuartile) [inline payment_acquirer_partner_link/models/payment_acquirer.py]:
```suggestion
# Copyright 2021-2022 Quartile Limited
```

File: payment_acquirer_partner_link/models/res_partner.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Quartile Limited
```
Comment (tack-yabeQuartile) [inline payment_acquirer_partner_link/models/res_partner.py]:
```suggestion
# Copyright 2021-2022 Quartile Limited
```

File: website_sale_partner_acquirer/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Quartile Limited
```
Comment (tack-yabeQuartile) [inline website_sale_partner_acquirer/__manifest__.py]:
```suggestion
# Copyright 2021-2022 Quartile Limited
```

File: website_sale_partner_acquirer/controllers/main.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Quartile Limited
```
Comment (tack-yabeQuartile) [inline website_sale_partner_acquirer/controllers/main.py]:
```suggestion
# Copyright 2021-2022 Quartile Limited
```

**Review by tack-yabeQuartile — APPROVED** [review]
LGTM, I fix Copyright

**Review by tack-yabeQuartile — APPROVED** [review]
LGTM

**Pull request merged**